src/database/schema/ChatSchema.ts

```diff
@@ -355,7 +359,7 @@ class ChatSchema implements BaseDbSchema {
       console.log(e);
     }
     try {
-      if (json?.message_type !== 'reply_prompt') {
+      if (isSavingAttachmentAllowed(json?.message_type)) {
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

Again, the condition for saving attachments has been refactored into a separate function `isSavingAttachmentAllowed`.
```diff
- if (json?.message_type !== 'reply_prompt') {
+ if (isSavingAttachmentAllowed(json?.message_type)) {
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->